MainRepository/Program.cs

```diff
@@ -1,7 +1,185 @@
+                        Console.WriteLine (new string (' ', Console.WindowWidth));
+                        break;
+                     }
+                     if (lkey.Key == ConsoleKey.Enter && !ValidWord ()) Console.WriteLine ("Not a valid word");
+                     if (lkey.Key == ConsoleKey.Backspace) {
+                        mWordle[i, 4] = mCircle;
+                        mWordle[i + 1, 0] = mDot;
+                        j--;
+                        mList.RemoveAt (4);
+                        break;
+                     }
+                  }
+               }
+               DisplayBoard (word);
+               if (mList.Count == 5) {
+                  DisplayBoard (word, true);
+                  if (mGreenList.Count == 5) {
+                     mGameOver = true;
+                     mCount++;
+                     return;
+                  }
+                  if (!ValidWord ()) j--;
+                  else mList.Clear ();
+               }
+            }
+         }
+         mCount++;
+      }
+   }
+
+   /// <summary>Set the foreground color for the list</summary>
+   public char Color (char clr) {
+      if (mGreenList.Contains (clr)) Console.ForegroundColor = ConsoleColor.Green;
+      else if (mYellowList.Contains (clr))
+         Console.ForegroundColor = ConsoleColor.Yellow;
+      else Console.ForegroundColor = ConsoleColor.Gray;
+      return clr;
+   }
+
+   /// <summary>Displays the current state of the board</summary>
+   public void DisplayBoard (string word, bool isEntered = false) {
+      Console.CursorLeft = 0;
+      Console.CursorTop = 0;
+      Console.CursorVisible = false;
+      Console.WriteLine ();
+      Console.CursorLeft = (Console.WindowWidth / 2) - 5;
+      for (int i = 0; i < 6; i++) {
+         for (int j = 0; j < 5; j++) {
+            if (i < mRow || isEntered) {
+               if (word[j] == mWordle[i, j]) {
+                  Console.ForegroundColor = ConsoleColor.Green;
+                  if (!mGreenList.Contains (mWordle[i, j]))
+                     mGreenList.Add (mWordle[i, j]);
+               } else if (word.Contains (mWordle[i, j])) {
+                  Console.ForegroundColor = ConsoleColor.Yellow;
+                  if (!mYellowList.Contains (mWordle[i, j]))
+                     mYellowList.Add (mWordle[i, j]);
+               } else {
+                  Console.ForegroundColor = ConsoleColor.Gray;
+                  if (!mGrayList.Contains (mWordle[i, j]))
+                     mGrayList.Add (mWordle[i, j]);
+               }
+            }
+            Console.Write (mWordle[i, j] + "  ");
+            Console.ResetColor ();
+         }
+         Console.WriteLine ();
+         Console.WriteLine ();
+         Console.CursorLeft = (Console.WindowWidth / 2) - 5;
+      }
+      PrintAlphabet ();
+   }
+
+   /// <summary>Prints the alphabet</summary>
+   public void PrintAlphabet () {
+      Console.CursorLeft = (Console.WindowWidth / 2) - 13;

```

Comment:
Repeatedly computing Console.WindowWidth / 2 instead save it in a variable and use it.